data/shortcodes.csv

```diff
@@ -20,7 +20,9 @@ Lausanne range - 0100 to 07FF,,,
 0114,elites-cio,Les membres du CIO de 1894 à 1972 - prosopographie,
 0115,activites-cs,Etude des activités de la Cinémathèque suisse entre 1951 et 1981,
 0116,medframes,"The Medieval Frame of Monumental Portal Sculpture (Pedestals, Canopies, Niches) : Cultural Transfers and Agency (12th-13th Centuries)",
-
+0117, no title available,
```

Comment:
why keeping this number if we don't need to add this project at the moment? I would remove this line